### ISSUE TITLE
Pod Install issue

### ISSUE THREAD

**Issue by nwest** [orta/cocoapods-keys#108]
First time trying cocoapods-keys. 

Gemfile
```rb
source 'https://rubygems.org'

gem 'cocoapods', '0.39.0.beta.4'
gem 'cocoapods-keys', '~> 1.5.0'
```

Podfile
```rb
plugin 'cocoapods-keys', {
  project: 'SomeProject',
  keys: [
    'SomeAPIKey'
  ]
}
```

During `pod install`:
```
NoMethodError - undefined method `*' for nil:NilClass
/Users/nwest/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-keys-1.5.1/lib/key_master.rb:32:in `generate_data'
/Users/nwest/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-keys-1.5.1/lib/key_master.rb:20:in `initialize'
/Users/nwest/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-keys-1.5.1/lib/plugin.rb:34:in `new'
/Users/nwest/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-keys-1.5.1/lib/plugin.rb:34:in `setup'
/Users/nwest/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-keys-1.5.1/lib/plugin.rb:98:in `install!'
/Users/nwest/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0.beta.4/lib/cocoapods/command/project.rb:71:in `run_install_with_update'
/Users/nwest/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0.beta.4/lib/cocoapods/command/project.rb:101:in `run'
/Users/nwest/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/claide-0.9.1/lib/claide/command.rb:312:in `run'
/Users/nwest/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0.beta.4/lib/cocoapods/command.rb:48:in `run'
/Users/nwest/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-0.39.0.beta.4/bin/pod:44:in `<top (required)>'
bin/pod:16:in `load'
bin/pod:16:in `<main>'
```

This doesn't happen on subsequent `pod install`s.

**Comment by orta** [comment]
Thanks!

**Comment by dpoggi** [comment]
So when https://github.com/orta/cocoapods-keys/blob/6e6c06b2477e950c07b0a8210645503aa25deaf1/lib/key_master.rb#L32 grabs @keys.values, it looks like https://github.com/orta/cocoapods-keys/blob/6e6c06b2477e950c07b0a8210645503aa25deaf1/lib/keyring.rb#L55 is going to feed it the nils osx_keychain returns for any keys that aren't yet in the keychain. Maybe this would work, unless empty strings would cause issues elsewhere?

    def keychain_value(key)
      ENV[key] || keychain[self.class.keychain_prefix + name, key] || ""
    end

**Comment by segiddins** [comment]
Isn't `verify_keychain_integrity` supposed to guard against that?

**Comment by orta** [comment]
released 1.5.2 with this and #110
